docs/java/features/odata/type-safe-client-odata-v2.mdx

```diff
@@ -413,20 +413,24 @@ try { ... }
 }
 ```
 
-This handling is the most generic, handling all possible failures. For more specific information there are dedicated exceptions inheriting from `ODataException`. Please tend to the [documentation](https://help.sap.com/doc/b579bf8578954412aea2b458e8452201/1.0/en-US/com/sap/cloud/sdk/datamodel/odata/client/exception/ODataException.html) for all the exception types.
+This handling is the most generic, handling all possible failures.
+For more specific information there are dedicated exceptions inheriting from `ODataException`. Please tend to the [documentation](https://help.sap.com/doc/b579bf8578954412aea2b458e8452201/1.0/en-US/com/sap/cloud/sdk/datamodel/odata/client/exception/ODataException.html) for all the exception types.
```

Comment:
Does this Javadoc link refer always to the latest version?

Reply:
It should, yes. But there is no guarantee. Currently it points to 3.30 but I think 3.31 wasn't uploaded yet. I haven't checked.